src/Pact/Typechecker.hs

```diff
@@ -486,7 +487,7 @@ assocTy :: TcId -> TypeVar UserType -> Type UserType -> TC ()
 assocTy ai av ty = do
   aty <- resolveTy =<< lookupTypes "assocTy" ai av
   debug $ "assocTy: " ++ show (av,aty) ++ " <=> " ++ show ty
-  unifyTypes' ai aty ty $ \r -> case r of
+  unifyTypes' ai aty ty $ \case
```

Comment:
Small nit, let's not start lambdacasing in this module just for this change.

Reply:
Ah, looks like one slipped in. One sec.